source/includes/dev/_microservice_dev_guide.md

```diff
@@ -195,6 +200,88 @@ Running tests under Docker is similar to running the actual microservice. The ea
 ./dataset.sh test
 ```
 
+#### Common errors and pitfalls
```

Comment:
This section is duplicated, no?

Reply:
no, there are different ways to handle things for docker users vs native users.

Reply:
What about the `mymachine` entry in /etc/hosts ?

Reply:
CT doesn't like endpoint changes. If you change routes in your microservice you might have to delete control tower and the associated docker data at ~/docker/control-tower to get things to work again

Reply:
You might run out of disc space due to too many docker images and data stored on you machine.
`docker system prune --all` removes all images, volumes and networks not currently in use and usually frees a lot of disk space this way. 
After running docker system prune, spinning up a docker container might take a little longer at first, b/c docker needs to download the images again.

Reply:
- On CT not liking endpoint changes: that should be handled by the cron I recently refactored. It is mentioned that, if you use CT, you should also run the cron.
- Ouf of disc space: IMO that's out of scope. As a rule of thumb, the docs should focus on what's specific to the API. In these sections, I've ignored that rule somewhat, for the sake of practicality, but the behavior you're describing is purely docker related, and can happen in any other docker project.